normandy/recipes/tests/api/v1/test_api.py

```diff
@@ -314,6 +322,10 @@ def test_list_filter_locales(self, api_client):
             for recipe in res.data:
                 assert recipe["id"] in [r1.id, r2.id]
 
+        def test_list_filter_locales_null_bytes(self, api_client):
+            res = api_client.get("/api/v1/recipe/?locales=\x00")
+            assert res.status_code == 400
```

Comment:
Can you also do a quick check on what the error that came back from the API is? It would be good to make sure that we are causing the desired error, and not something else.